electron/config.mjs

```diff
@@ -4,6 +4,25 @@ import { join, dirname, parse } from 'path';
 import log from 'electron-log/main.js';
 import util from 'util';
 import { renameSync } from 'fs';
+import os from 'os';
+
+function browserSettingsLocation() {
+  const homeDir = os.homedir();
+  let configDir;
+  if (os.platform() === 'darwin') {
+    configDir =
+      process.env.XDG_CONFIG_HOME ||
+      join(homeDir, 'Library', 'Application Support');
+  } else if (os.platform() === 'win32') {
+    configDir = join(homeDir, 'AppData', 'Local');
+  } else if (os.platform() === 'linux') {
+    configDir = process.env.XDG_CONFIG_HOME || join(homeDir, '.config');
+  } else {
+    throw new Error('Unsupported platform');
+  }
+
+  return join(configDir, 'acorn', 'browsersettings.json');
+}
```

Comment:
I hate how this function is duplicated from actions/browser.tsx. But I'm not sure if there is a way for me to define it in one place and have it be callable from both places. Please let me know if there is a way to do that.

Reply:
That I personally know of, no. I've had a lot of similar headaches. Ideally we'd make a global lib package that both the Next app and electron could import. To do that right though I think we'd need the socket server and config code to be in typescript so it isn't super simple in just this PR.